package.json

```diff
@@ -213,7 +213,9 @@
     "test:inspect": "yarn compile:test && node --inspect-brk node_modules/jasmine/bin/jasmine.js --config=jasmine.json",
     "test:lsp": "yarn compile:integration && jasmine --config=integration/lsp/jasmine.json",
     "test:e2e": "yarn compile:integration && node dist/integration/e2e",
-    "test:syntaxes": "yarn compile:syntaxes-test && yarn build:syntaxes && jasmine dist/syntaxes/test/driver.js"
+    "test:syntaxes": "yarn compile:syntaxes-test && yarn build:syntaxes && jasmine dist/syntaxes/test/driver.js",
+    "test:bazel": "yarn bazel test --test_tag_filters=unit_test //...",
+    "test:bazel-watch": "yarn ibazel test --test_tag_filters=unit_test //..."
```

Comment:
It looks like `yarn test` only includes 2 specs now. I think this was actually a result of the previous change. I'm fine with just updating the existing scripts instead of creating new ones.

So `test` would be the bazel test and `test:inspect` would be bazel test with `--config=debug`

Reply:
I could flip `yarn test` to use bazel in this, tho that would mean that CI would be using bazel as well since `test.sh` calls `yarn run test`. Might be preferable to run bazel CI in a parallel step to the legacy CI for some time before full cut over.

The `--config=debug` will only work if you are testing an individual target and the jasmine unit tests are now split across three `//common/tests:test` `//server/src/tests:test` and `//client/src/tests:test`.

So when setting `--config=debug` you have to pick which individual target you want to run.

For example `bazel test --config=debug //server/src/tests:test`:

```
$ bazel test --config=debug //server/src/tests:test
WARNING: Streamed test output requested. All tests will be run locally, without sharding, one at a time
INFO: Analyzed target //server/src/tests:test (0 packages loaded, 2 targets configured).
INFO: Found 1 test target...
Debugger listening on ws://127.0.0.1:9229/1cf7a13b-7c85-4fb1-b892-b5c8d545d2ae
For help, see: https://nodejs.org/en/docs/inspector
```

This wouldn't fit in with the current single `test:inspect` script. Perhaps break it out into `test:inspect-common` `test:inspect-server` and `test:inspect-client`?